### PR TITLE
DSA 1.1.2.2 Array Rearrangement STL #446

### LINKED ISSUE #446: DSA 1.1.2.2 Array Rearrangement STL
## Description
Minimum swaps required to bring elements less than equal to k together, STL implementations of rearrangement(sort comparator etc).

## Task
- [ ] Documentation
- [ ] Audio
- [ ] Video

## Note
- While requesting the issue to be assigned, please mention your **NAME & BATCH NUMBER**.
- Kindly refer to [Issue Guidelines](https://github.com/girlscript/winter-of-contributing/blob/DSA/DSA/CONTRIBUTING.md#issues).
- In case you see that the issue has already been assigned, try to request for a different task within the same issue.
- Changes should be made inside the ````DSA/directory```` & ````DSA```` branch.
- Name of the file should be same as the issue.
- This issue is only for 'GWOC' contributors of 'DSA' Domain.

### PR DESCRIPTION
**Pull request by geeky01adarsh**
<hr>

## Description 📜

Prepared documentation of Array Rearrangement STL.
Problem Statement :- Minimum swaps required to bring elements less than equal to k together, STL implementations of rearrangement(sort comparator etc).

Fixes #446 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [x] DSA
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I'm GWOC'21 contributor

<hr>

@pjdurden sir, can you please check it once.

<hr>

### CODE REVIEW

**Review by japneetbhatia — APPROVED** [review]

**Comment by rohit-goyal** [comment]
Hello My name is Rohit Goyal 
pls assign this issue to me

**Comment by geeky01adarsh** [comment]
> Hello My name is Rohit Goyal
> pls assign this issue to me

This is a PR not an issue.

**Comment by geeky01adarsh** [comment]
@nimisha-yadav 
Mam, can you check this once? Please!!!

File: DSA/1.1 Arrays/1.1.2.2_Array_Rearrangement_STL.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+```
+
+
+### Java implementation
+```
+
+import java.lang.*;
+import java.util.*;
+class GFG {
```
Comment (nimisha-yadav) [inline DSA/1.1 Arrays/1.1.2.2_Array_Rearrangement_STL.md:86]: You did not even change the class name.

**Comment by nimisha-yadav** [comment]
Plagiarized code from here: https://www.geeksforgeeks.org/minimum-swaps-required-bring-elements-less-equal-k-together/

**Pull request closed without merging**